src/atomate2/vasp/sets/matpes.py

```diff
@@ -7,93 +7,32 @@
 from __future__ import annotations
 
 from dataclasses import dataclass, field
-from importlib.resources import files as get_mod_path
-from typing import TYPE_CHECKING
 
-from monty.serialization import loadfn
+from pymatgen.io.vasp.sets import MatPESStaticSet
 
 from atomate2.vasp.sets.base import VaspInputGenerator
 
-if TYPE_CHECKING:
-    from pymatgen.core import Structure
-    from pymatgen.io.vasp import Outcar, Vasprun
-
-
-# POTCAR section comes from PARENT but atomate2 does not support inheritance yet
-_BASE_MATPES_PBE_STATIC_SET_NO_POTCAR = loadfn(
-    get_mod_path("pymatgen.io.vasp") / "MatPESStaticSet.yaml"
-)
-_POTCAR_BASE_FILE = f"{_BASE_MATPES_PBE_STATIC_SET_NO_POTCAR['PARENT']}.yaml"
-_POTCAR_SET = loadfn(get_mod_path("pymatgen.io.vasp") / _POTCAR_BASE_FILE)
-_BASE_MATPES_PBE_STATIC_SET = {**_POTCAR_SET, **_BASE_MATPES_PBE_STATIC_SET_NO_POTCAR}
+_matpes_config_no_magmom = MatPESStaticSet.CONFIG
+_matpes_config_no_magmom["INCAR"].pop("MAGMOM")
 
```

Comment:
> MatPES set (@janosh and @shyuep): The MatPES magmoms were missing from the atomate2 set previously, but are present in their pymatgen set. For now, the atomate2 MatPES set removes the magmoms for consistency with previous versions, but that may have been a bug

that sounds like a bug! thanks for catching this! let's remove the `.pop("MAGMOM")`

Reply:
Done!